src/app/drive/components/MoveItemsDialog/MoveItemsDialog.tsx

```diff
@@ -81,10 +81,9 @@ const MoveItemsDialog = (props: MoveItemsDialogProps): JSX.Element => {
   }, [newFolderIsOpen]);
 
   const onShowFolderContentClicked = (folderId: number, name: string): void => {
-    dispatch(fetchFolderContentThunk(folderId))
+    dispatch(fetchDialogContentThunk(folderId))
       .unwrap()
       .then(() => {
-        setIsLoading(true);
```

Comment:
With what you propose, everything continues to be reloaded with each action. I have moved it to useEffect so that it only reloads when the modal is opened.